tutorials/quick_start/pytorch_fw/ultralytics_lib/replacers.py

```diff
@@ -22,49 +22,41 @@
 import torch
```

Comment:
I'm a bit bothered by the file names.. It seems odd that we have pose_replacers.py, segment_replacers.py and we don't have detect_replacers.py. not sure it's critical..

Reply:
I'll follow this comment to ask: why the a separation between "replaces" and all other task-specific replacer files?
From my perspective just from looking at the files name, I wouldn't know what to expect to find under "replaces" that would not appear in the task-specific file.
Maybe change replacers.py file name to something more indicative that doesn't collide with the other files.

Reply:
There is a detect_replacers, maybe common_replacers is a better name? or yolo_replacers?